libs/providers/config-cat-web/package.json

```diff
@@ -0,0 +1,13 @@
+{
+  "name": "@openfeature/config-cat-web-provider",
+  "version": "0.6.1",
+  "scripts": {
+    "publish-if-not-exists": "cp $NPM_CONFIG_USERCONFIG .npmrc && if [ \"$(npm show $npm_package_name@$npm_package_version version)\" = \"$(npm run current-version -s)\" ]; then echo 'already published, skipping'; else npm publish --access public; fi",
+    "current-version": "echo $npm_package_version"
+  },
+  "peerDependencies": {
+    "@openfeature/web-sdk": "^1.0.0",
+    "@openfeature/config-cat-core": "0.1.0",
```

Comment:
You may want to consider removing this, though it's up to you.

`updateBuildableProjectDepsInPackageJson` will automatically add `@openfeature/config-cat-core` as a dep, making things easy for users, and in this case, I don't think you _need_ a peer dep, since all of the usage of `@openfeature/config-cat-core` is internal to this library and none of it's surface area is exposed to users.

The only time you really need peer deps is if you need to ensure the library version and a version installed by consumers matches, but this isn't applicable here as far as I can see.

Reply:
This is a good point and I will use it!🙂
Will change it.